content/docs/tasks/traffic-management/secure-ingress/index.md

```diff
@@ -445,6 +445,80 @@ In this subsection, perform the same steps as in the [Generate client and server
         `"""`
     {{< /text >}}
 
+## Configure end-user authentication on ingress gateway
+
+The Istio ingress gateway supports end-user authentication by setting up a JWT
```

Comment:
Passive voice issue. To fix, I suggest:
To support end-user authentication, the Istio ingress gateway sets up a JWT

Reply:
thanks, updated.